.codecov.yaml

```diff
@@ -0,0 +1,10 @@
+coverage:
+  status:
+    project:
+      default: off
+      test:
+        target: 0%
+        flags: test
+      integration:
+        target: 0%
```

Comment:
is this going to get bumped up later?

Reply:
Yep, I have a follow-up task to do just that.